go.mod

```diff
@@ -1,13 +1,17 @@
 module github.com/duo-labs/webauthn
 
+go 1.17
```

Comment:
this will set min required version to 1.17 - that's not good for a library!

Reply:
Setting `go 1.17` does not prevent users with earlier Go versions from successfully building packages from this module since we are not using any Go 1.17 specific features yet.

In addition, Go 1.16 will soon reach its end-of-life when Go 1.18 is released in February 2022.